_posts/2022-12-6-uma_outreachy.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Outreachy Report - 2022 Improve MDAnalysis by implementing type hinting
+---
+
+## Motivation 
+
+The object-oriented Python library MDAnalysis analyzes trajectory data derived from molecular dynamics (MD) simulations in many popular formats.
+Python's dynamic nature enables us to develop with speed, flexibility, and ease of use, but if you are not careful, you can result in short-term expedience at the expense of long-term maintainability as a result of the dynamic nature of Python.
+
+While type hints and type annotations do hint towards or indicate the appropriate types they do not enforce them. By utilizing typecheckers such as mypy, I made sure that the code is performing what it should regarding the types passed around between functions, and the annotated function signatures only boosted the readability of the code as well as improved communication within it.
```

Comment:
Adding links to docs on type hinting in Python would be really useful. Where should someone interested in doing this go from your post?